src/app/store/reducers/index.ts

```diff
@@ -0,0 +1,8 @@
+import { ActionReducer, ActionReducerMap, createFeatureSelector, createSelector, MetaReducer } from '@ngrx/store';
+import { environment } from '../../../environments/environment';
+
+export interface State {}
```

Comment:
Setup for the global app state (as opposed to feature state.) Seems like most of what we're doing belongs in feature state, not sure yet what we'll be putting in global state (besides the route from router-store.)